### PR TITLE
add project configurations

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: configuration/pipeline_config.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      ],
+      "TestContactUUIDs": [
+        "94075043-57c6-43c7-8b89-880478ccad8e",
+        "1b895d33-a592-472f-9d01-dbf9b412a6e1"
+      ]
+    }
+  ],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "UNICEF_KENYA_phone_number_avf_phone_id"
```
Comment (as2388) [inline configuration/pipeline_config.json:27]: Is this supposed to be different from the project name?

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  },
+  "RapidProKeyRemappings": [
+    {"RapidProKey": "avf_phone_id", "PipelineKey": "uid"},
+    
+    {"RapidProKey": "Rqa_S01E01 (Text) - unicef_covid19_ke_s01e01_activation", "PipelineKey": "rqa_s01e01_raw", "IsActivationMessage": true},
+    {"RapidProKey": "Rqa_S01E01 (Run ID) - unicef_covid19_ke_s01e01_activation", "PipelineKey": "rqa_s01e01_run_id"},
+    {"RapidProKey": "Rqa_S01E01 (Time) - unicef_covid19_ke_s01e01_activation", "PipelineKey": "sent_on"},
+
+    {"RapidProKey": "Rqa_S01E01 (Text) - unicef_covid19_ke_s01e02_activation", "PipelineKey": "rqa_s01e02_raw", "IsActivationMessage": true},
```
Comment (as2388) [inline configuration/pipeline_config.json]: copy/past error

**Review by as2388 — APPROVED** [review]

File: configuration/pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: updated thanks

File: configuration/pipeline_config.json (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json:27]: Have used the textit instance name with a forwarding thinking design of if we get another UNICEF Kenya project we building up from this table.

**Pull request merged**